src/app/shared/services/record.service.ts

```diff
@@ -41,4 +45,12 @@ export class RecordService {
       .map(res => res.json());
   }
 
+  saveRecord(record: Object): Observable<Object> {
+    let postData = {
+      'pid_type': this.type,
+      'recid': this.recid,
+      'json_data': record
+    };
+    return this.http.post('http://localhost:5000/editor/save', postData).map(res => res.json());
```

Comment:
One thing to check is the prefix we set here https://github.com/inspirehep/record-editor/blob/master/src/app/app.module.ts#L29 I think it is for angular-router to base its routing always on that prefix, but would be good to check if it affects the http requests.